app/components/Utility/GatewaySelector.jsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import Translate from "react-translate-component";
+import Icon from "../../components/Icon/Icon";
+import {Link} from "react-router-dom";
+import {Select} from "bitshares-ui-style-guide";
+
+class GatewaySelector extends React.Component {
+    constructor() {
+        super();
+    }
+
+    render() {
+        let {
+            gatewayStatus,
+            selectedGateway,
+            error,
+            nAvailableGateways
+        } = this.props;
+
+        let selectGatewayList = 
```

Comment:
As this is a new class please add proper docs. E.g. here what the intent is of selectGatewayList and how this is filtered and sorted.

Reply:
I will document this better